source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -252,13 +274,14 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
       // Currently it is not possible to add {{"append": "1"}, {"append": "2"}} (the intended
       // combined headers: {{"original": "true"}, {"append": "1"}, {"append": "2"}}) to the request
       // to upstream server by only sets `headers_to_append`.
```

Comment:
Is this comment still true?

Reply:
That's a good question. Let me think about it. I think it can be achieved using `ADD_IF_ABSENT` now.